config/manager/manager.yaml

```diff
@@ -24,8 +24,16 @@ spec:
     spec:
       securityContext:
         runAsNonRoot: true
+      volumes:
+      - name: default-user-workload-pull-secret
+        secret:
+          secretName: default-user-workload-pull-secret
       containers:
-      - command:
+      - volumeMounts:
+        - name: default-user-workload-pull-secret
+          mountPath: "/etc/build"
```

Comment:
```suggestion
          mountPath: "/etc/build-credentials"
```

Reply:
🤔 why so?

Reply:
Well, to me `build` folder is more about storing current build artifacts (maybe a habit from `make`). Adding word `credentials` makes it clear.